modules/mw.cx.TargetArticle.js

```
@@ -216,52 +216,65 @@ mw.cx.TargetArticle.prototype.publishSuccess = function (response, jqXHR) {
+	// const mdwiki_result = publishResult.mdwiki_result || [];
+	const wikipedia_result = publishResult.wikipedia_result || [];
+
+	const wd_data = wikipedia_result.LinkToWikidata || publishResult.LinkToWikidata;
+
+	if (publishResult.wikipedia_result) {
 		console.log("_____");
-		console.log("local result: " + JSON.stringify(publishResult.save_result_all.result));
-		console.log("mdwiki_result: " + JSON.stringify(publishResult.save_result_all.mdwiki_result));
+		// console.log("local result: " + JSON.stringify(publishResult.local_result, null, 1));
+		const resultCopy = { ...wikipedia_result };
+		delete resultCopy.LinkToWikidata;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)